tdex/exports.sh

```diff
@@ -0,0 +1,7 @@
+export APP_TDEX_PORT="9090"
+
+daemon_hidden_service_file="${EXPORTS_TOR_DATA_DIR}/app-${EXPORTS_APP_ID}-daemon/hostname"
+export APP_TDEX_DAEMON_HIDDEN_SERVICE="$(cat "${daemon_hidden_service_file}" 2>/dev/null || echo "daemon_not_yet_set.onion")"
+
+# Necessary for the tdexd daemon to be able to write to the volume
+sudo chmod -R 777 "${UMBREL_ROOT}/app-data/tdex/tdex-data"
```

Comment:
Did you want to give full read/write/executable permissions to user, group, and others here? 
or does giving read/write/execute to user and read/execute to group and others work for you... for example, by running:
`sudo chmod -R 755 "${UMBREL_ROOT}/app-data/tdex/tdex-data"`

Reply:
I'm okay with this going out as-is and then we can consider handling this another way in the future to avoid giving such relaxed permissions to your app's data directory.

I quickly looked at the [dockerfile for tdex](https://github.com/tdex-network/tdex-daemon/blob/master/Dockerfile) and noticed that the user and group are set to `tdex`. One way you might be able to solve this issue is by having tdex run as user/group 1000.